design-proposals/README.md

```diff
@@ -101,3 +101,4 @@ Next KLIP number: **61**
 | KLIP-58: [`LIMIT` clause for Pull Queries](https://github.com/confluentinc/ksql/pull/8298)                                       | Approved |      0.24.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/8298) |
 | [KLIP-59: JSON functions](klip-59-json-functions.md)                                                                             | Approved |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/8550) | 
 | [KLIP-60: Support input topics with multiple message types](klip-60-multiple-message-types.md)                                                                             | Proposal |                   |            |  | 
+| [KLIP-61: Bootstrapped subscriptions](klip-61-bootstrapped-subscriptions.md)                                                     | Proposal |                   |            |  | 
```

Comment:
Can we call it `Bootstrapped TABLE subscriptions` (for clarify that is only applies to TABLES)?

Reply:
Sure that makes sense to me